examples/c/lsm.c

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+/* Copyright (c) 2024 dave
+ * Based on minimal.c by Facebook */
+#include <stdio.h>
+#include <unistd.h>
+#include <sys/resource.h>
+#include <bpf/libbpf.h>
+#include "lsm.skel.h"
+
+// Notice: Ensure your kernel version is 5.7 or higher, BTF (BPF Type Format) is enabled, 
+// and the file '/sys/kernel/security/lsm' includes 'bpf'.
+static int libbpf_print_fn(enum libbpf_print_level level, const char *format, va_list args)
+{
+	return vfprintf(stderr, format, args);
+}
+
+int main(int argc, char **argv)
+{
+	struct lsm_bpf *skel;
+	int err;
+
+	/* Set up libbpf errors and debug info callback */
+	libbpf_set_print(libbpf_print_fn);
+
+	/* Open BPF application */
+	skel = lsm_bpf__open();
+	if (!skel) {
+		fprintf(stderr, "Failed to open BPF skeleton\n");
+		return 1;
+	}
+
+	/* Load & verify BPF programs */
+	err = lsm_bpf__load(skel);
+	if (err) {
```

Comment:
you can combine this into a single `lsm_bpf__open_and_load()` call

Reply:
=> `lsm_bpf__open_and_load` done